pkg/schema/plugins.schema.json

```diff
@@ -11,41 +11,46 @@
       "description": "Version of the plugins schema to which this document adheres",
       "type": "string",
       "default": "1.0.0"
-    }
-  },
-  "required": [
-    "schemaVersion"
-  ],
-  "title": "Plugins json schema",
-  "type": "object",
-  "additionalProperties": {
-    "type": "object",
-    "properties": {
-      "key": {
-        "type": "string"
-      },
-      "value": {
+    },
+    "plugins": {
+      "description": "The definition of plugins to install",
+      "type": "object",
+      "default": "1.0.0",
```

Comment:
This default shouldn't be defined here. I assume you meant to define it directly on schemaVersion?